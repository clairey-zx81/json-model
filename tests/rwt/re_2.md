- schema: format is not implemented/used by the schema checker
