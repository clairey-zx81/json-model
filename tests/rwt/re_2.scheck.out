re_2.0.true.json: True
re_2.0.false.json: False
re_2.1.false.json: False
